plugins/templates/getPipelineUsage.js

```diff
@@ -0,0 +1,51 @@
+'use strict';
+
+const boom = require('@hapi/boom');
+const joi = require('joi');
+const schema = require('screwdriver-data-schema');
+const nameSchema = schema.models.template.base.extract('name');
+const versionSchema = schema.models.template.base.extract('version');
+const tagSchema = schema.models.templateTag.base.extract('tag');
+const getSchema = schema.api.pipelineUsage.get;
```

Comment:
Per https://github.com/screwdriver-cd/data-schema/pull/527#pullrequestreview-1590187795
```suggestion
const getSchema = schema.api.pipelineUsage.output;
```

Reply:
I think `get` better matches the naming format we've been using for other files.

Reply:
Not pushy on either of these (`get` vs `output`).
Here is my observation
1. All the files under [`data-schema/models`](https://github.com/screwdriver-cd/data-schema/tree/master/models) directory represents an entity like `pipeline`, `job` etc
     1. Each entity is bound to a table in database
     2. We can perform CRUD operations on these. Hence we have standardized schema [`get`, `create`, `update`] corresponding to each operation, which will be used in API endpoints to validate input/output
2. All the files under [`data-schema/api`](https://github.com/screwdriver-cd/data-schema/tree/master/api) directory does not represent an entity
     1. These are not bound to a table in database
     2. It does support all the CRUD operations. It is focused to an API endpoint. Hence majority of the files have `input` and `output` schema which is used to validate input and/or output of a specific endpoint.

Since the schema under discussion is being added under `data-schema/api`, suggestion was made to follow the existing convention to name the schema as `output`.

From another perspective, this schema is a kinda subset of `get` schema defined in[`data-schema/models/pipeline.js`](https://github.com/screwdriver-cd/data-schema/blob/master/models/pipeline.js).
We could add the new schema in `data-schema/blob/master/models/pipeline.js` and name it something like `getForTemplateUsageReport`